src/Jackett.Common/Definitions/eltorrents.yml

```diff
@@ -0,0 +1,981 @@
+---
+  site: Eltorrents
+  name: Eltorrents
+  description: "Eltorrents is a Public tracker, a  popular alternative to the original ET site, providing Movie / TV / General magnets"
+  language: en-us
+  type: public
+  encoding: UTF-8
+  links:
+    - https://eltorrents.com/
+  certificates:
+    - 2d42cf7a33712d4798d6ba6536824fb448919889 # incomplete CA chain
```

Comment:
certificates look fine for the page, this shouldn't be needed

Reply:
Ok, when I tried I got some error, maybe I did some wrong operation at beginning...
No problem if not needed we always can add it later :D